.mise/tasks/setup

```diff
@@ -160,6 +161,10 @@ install_anvil() {
   echo -e "Foundry installation done"
 }
 
+setup_rust_target(){
+
+}
```

Comment:
Is this meant to be empty? I'm a bit unsure on what this does

Reply:
already fixed this in a later commit (directly to main)